conf/application.conf

```diff
@@ -62,6 +62,11 @@ microservice {
               port = 8701
       }
 
+    address-lookup-frontend {
```

Comment:
can we make an entry in app config base and also app config qa, staging and prod. see other services on how we override the host